internal/controllers/reconciliation/controller.go

```diff
@@ -88,6 +89,16 @@ func (c *Controller) Reconcile(ctx context.Context, req *reconstitution.Request)
 		prev, _ = c.resourceClient.Get(ctx, compRef, &req.Resource)
```

Comment:
Unrelated to this PR, but is it safe to ignore the error here?

Reply:
Yes because we assume that `prev` can be a nil pointer later on. Some folks think this is a bad practice, I prefer it over passing around a bool. 🤷